docs/files.md

```diff
@@ -746,3 +747,35 @@ A file's watermark can be removed by calling
 ```js
 client.files.removeWatermark('67890', callback);
 ```
+
+Get Representation
+------------------
+
+A file's representation can be retrieved by calling
+[`files.getRepresentationInfo(fileID, representationTypes, options,
+callback)`](https://opensource.box.com/box-node-sdk/Files.html#getRepresentationInfo).
+You will be able to fetch a pdf representation, thumbnail representation, multi-page images
+representation, and fetching text representation.
+
+You can fetch different dimensions for the representation by calling
+```js
+client.files.getRepresentationInfo('67890', client.files.representation.IMAGE_LARGE, null, callback);
+```
+Additionally there are two header options you can include. set_content_disposition_type and set_content_disposition_filename.
+
+set_content_disposition_type must be set to either 'inline' to try
+and open the representation within the browser or 'attachment' to force the browser to download the
```

Comment:
I still think we need to get more clarity about exactly what these parameters mean for someone using the Node SDK.  If you call with one vs. the other, what is the actual difference in behavior?  I think talking about the browser here might confuse users who aren't in a browser.

I'm also really confused about what these mean for this specific endpoint.  This is not the call for the actual asset, it's for the JSON object describing the representations.  Do these headers even apply here, or are they for the actual asset request itself?